Usbipd/Usbipd.csproj

```diff
@@ -36,6 +36,9 @@ SPDX-License-Identifier: GPL-2.0-only
   </ItemGroup>
 
   <ItemGroup>
+    <None Update="usb.ids">
+      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
+    </None>
```

Comment:
Not for this PR (i.e., we can do that in a separate PR), but I like the auto-updater in https://github.com/woodruffw/usb-ids.rs/blob/main/.github/workflows/usbids-updater.yml
Of course, they got the licensing all wrong (!), but we are doing the right thing (GPLv2).